x/axelarnet/keeper/migrate.go

```diff
@@ -1,10 +1,43 @@
 package keeper
 
-import sdk "github.com/cosmos/cosmos-sdk/types"
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
 
-// GetMigrationHandler returns the handler that performs in-place store migrations
-func GetMigrationHandler() func(_ sdk.Context) error {
+	"github.com/axelarnetwork/axelar-core/utils"
+	"github.com/axelarnetwork/axelar-core/x/axelarnet/types"
+)
+
+// GetMigrationHandler returns the handler that performs in-place store migrations from v0.22 to v0.23. The
+// migration includes
+// - set existing IBC transfers status to completed
+// - remove transfer under failedTransfer prefix
+// - remove nonce
+func GetMigrationHandler(k Keeper) func(_ sdk.Context) error {
 	return func(ctx sdk.Context) error {
+		setIBCTransfersCompleted(ctx, k)
```

Comment:
Are we sure this is ok? Even those that are pending are gonna be set as completed now cuz the queue is also gone.

Reply:
no other good ways, we need to clear all pending transfer. and since migration only affects testnet, I am 'ok' to take some risk.